kernelci/cli/base.py

```diff
@@ -582,7 +570,7 @@ def parse_args_with_parser(parser, glob):
     *parser* is an `ArgumentParser` instance used to parse the command line.
```

Comment:
`args` parameter description missing in docstring

Reply:
Fixed